packages/channel-client/src/types.ts

```diff
@@ -3,34 +3,25 @@ import {
   ChannelResult,
   Participant,
   Allocation,
-  SiteBudget
+  SiteBudget,
+  Message
 } from '@statechannels/client-api-schema';
 
-export interface Message<T = object> {
-  recipient: string; // Identifier of user that the message should be relayed to
-  sender: string; // Identifier of user that the message is from
-  data: T; // Message payload. Format defined by wallet and opaque to app.
-  // But useful to be able to specify, for the purposes of the fake-client
-}
-
 export type UnsubscribeFunction = () => void;
 
-// The message Payload is designed to be opaque to the app. However, it's useful
-// to be able to specify the Payload type for the FakeChannelClient, as we'll be
-// manipulating it within the client.
-export interface ChannelClientInterface<Payload = object> {
+export interface ChannelClientInterface {
   /*
     Queuing a message is meant for when the app receives messages from
     the wallet meant for the opponent's app (and hence the opponent's wallet).
   */
-  onMessageQueued: (callback: (message: Message<Payload>) => void) => UnsubscribeFunction;
+  onMessageQueued: (callback: (message: Message<object>) => void) => UnsubscribeFunction;
```

Comment:
I think we should change `Message` so it doesn't accept a type, and the `payload` is of type `Unknown`.

Reply:
d1c659f